Sources/MongoSwift/BSON/BSONDecoder.swift

```diff
@@ -109,7 +109,24 @@ public class BSONDecoder {
     }
 
     /// Initializes `self`.
-    public init() {}
+    public init(options: CodingStrategyProvider? = nil) {
+        self.configureWithOptions(options: options)
+    }
+
+    /// Initializes `self` by copying the options of another `BSONDecoder`.
```

Comment:
can we add a sentence to this docstring clarifying what options take precedence when `other` conflicts with `options`?

Reply:
done